docs/developers/nft-tokens.md

```diff
@@ -0,0 +1,400 @@
+          "@" + <"canFreeze" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded> +
+          "@" + <"canWipe" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded> +
+          "@" + <"canPause" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded> +
+          "@" + <"canTransferNFTCreateRole" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded>
+}
+```
+
+The receiver address `erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u` is a built-in system smart contract (not a VM-executable contract), which only handles token issuance and other token management operations, and does not handle any transfers.
+The contract will add a random string to the ticker thus creating the **token identifier**. The random string starts with “-” and has 6 more random characters. For example, a token identifier could look like _ALC-6258d2_.
+
+## **Parameters format**
+
+Token Name:
+
+- length between 3 and 20 characters
+- alphanumeric characters only
+
+
+Token Ticker:
+
+- length between 3 and 10 characters
+- alphanumeric UPPERCASE only
+
+## **Issuance examples**
+
+For example, a user named Alice wants to issue an ESDT called "AliceTokens" with the ticker "ALC". The issuance transaction would be:
+
+```
+IssuanceTransaction {
+    Sender: erd1sg4u62lzvgkeu4grnlwn7h2s92rqf8a64z48pl9c7us37ajv9u8qj9w8xg
+    Receiver: erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u
+    Value: 5000000000000000000
+    GasLimit: 60000000
+    Data: "issueSemiFungible" +
+          "@416c696365546f6b656e73" +
+          "@414c43" +
+}
+```
+Once this transaction is processed by the Metachain, Alice becomes the designated **manager of AliceTokens**, and is granted a balance of 4091 AliceTokens, to do with them as she pleases. She can increase the total supply of tokens at a later time if needed. For more operations available to ESDT token managers, see [Token management](/developers/esdt-tokens#token-management).
+
+If the issue transaction is successful, a smart contract result will mint the requested token and supply in the account used for issuance, which is also the token manager.
+In that smart contract result, the `data` field will contain a transfer syntax which is explained below. What is important to note is that the token identifier can be fetched from
+here in order to use it for transfers. Alternatively, the token identifier can be fetched from the API (explained also below).
+
+## **Roles** ##
+
+In order to be able to perform actions over a token, one needs to have roles assigned.
+The existing roles are:
+
+- ESDTRoleNFTCreate : this role allows one to create a new NFT/SFT
+- ESDTRoleNFTAddQuantity : this role allows one to add quantity of a specific NFT/SFT
+- ESDTRoleNFTBurn : this role allows one to burn quantity of a specific NFT/SFT
+
+To see how roles can be assigned, please here to [this](/developers/nft-tokens#assigning-roles) section.
```

Comment:
please **here**? to this section.

maybe "refer"?

Reply:
:1st_place_medal: done